test/pkg/steps/data.go

```diff
@@ -36,13 +36,25 @@ var (
 	logsKubernetesObjects = []runtime.Object{&imgv1.ImageStreamList{}, &v1beta1.KogitoRuntimeList{}, &v1beta1.KogitoBuildList{}, &v1beta1.KogitoSupportingService{}, &v1beta1.KogitoInfraList{}, &olmapiv1alpha1.ClusterServiceVersionList{}}
 )
 
+type thirdPartyProject struct {
```

Comment:
oh I was confusing in what I said yesterday I guess.
You can directly define a type to be a string instead of having a struct. See https://github.com/kiegroup/kogito-operator/blob/master/api/kogitoinfra_types.go#L24 as an example

Reply:
oh, ok, sorry, that is my low skills in go, will fix it